Libraries/LibGUI/IconView.cpp

```diff
@@ -523,24 +523,33 @@ void IconView::paint_event(PaintEvent& event)
 
         auto font = font_for_index(item_data.index);
 
-        painter.fill_rect(item_data.text_rect, background_color);
+        Gfx::IntRect text_rect = item_data.text_rect;
+        auto icon_translation = translation.y() - 12;
```

Comment:
I need to work out a better way of finding the y distance between two icons. This seems to work currently, but I don't know if it will break things down the line.